typescript/cli/tests/warp.zs-test.ts

```diff
@@ -0,0 +1,138 @@
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { Wallet } from 'ethers';
+import { $ } from 'zx';
+
+import { WarpCoreConfig, WarpRouteDeployConfig } from '@hyperlane-xyz/sdk';
+import { Address, assert, eqAddress } from '@hyperlane-xyz/utils';
+
+import { HypNative__factory } from '../../../solidity/dist/factories/contracts/token/HypNative__factory.js';
```

Comment:
I'd expect this to be exported from `@hyperlane-xyz/core`. Is the direct import path necessary?